<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideBar</class>
 <widget class="QWidget" name="SideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>80</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>80</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lab_rate">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>50M/10G</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QProgressBar" name="progress_bar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>3</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>3</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar {
    background-color: #ecf0f9;  /* 背景颜色 */
}

QProgressBar::chunk {
    background-color: #2660ff;  /* 已完成部分的颜色 */

}
</string>
           </property>
           <property name="value">
            <number>25</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QToolButton" name="btn_file">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    background-color: rgb(245, 247, 250); /* 默认背景颜色 */
    border: none; /* 去掉边框 */
    border-radius: 10px; /* 圆角按钮 */
    padding: 5px;
    color: black; /* 默认字体颜色 */
}

QToolButton:hover {
    background-color: rgb(235, 237, 241); /* 鼠标悬浮时的背景颜色 */
}
</string>
         </property>
         <property name="text">
          <string>文件</string>
         </property>
         <property name="icon">
          <iconset resource="../../resource/icon/icon.qrc">
           <normaloff>:/SideBar/folder_common.svg</normaloff>:/SideBar/folder_common.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_transfer">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    background-color: rgb(245, 247, 250); /* 默认背景颜色 */
    border: none; /* 去掉边框 */
    border-radius: 10px; /* 圆角按钮 */
    padding: 5px;
    color: black; /* 默认字体颜色 */
}

QToolButton:hover {
    background-color: rgb(235, 237, 241); /* 鼠标悬浮时的背景颜色 */
}
</string>
         </property>
         <property name="text">
          <string>传输</string>
         </property>
         <property name="icon">
          <iconset resource="../../resource/icon/icon.qrc">
           <normaloff>:/SideBar/transfer_common.svg</normaloff>:/SideBar/transfer_common.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_share">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    background-color: rgb(245, 247, 250); /* 默认背景颜色 */
    border: none; /* 去掉边框 */
    border-radius: 10px; /* 圆角按钮 */
    padding: 5px;
    color: black; /* 默认字体颜色 */
}

QToolButton:hover {
    background-color: rgb(235, 237, 241); /* 鼠标悬浮时的背景颜色 */
}
</string>
         </property>
         <property name="text">
          <string>分享</string>
         </property>
         <property name="icon">
          <iconset resource="../../resource/icon/icon.qrc">
           <normaloff>:/SideBar/share_common.svg</normaloff>:/SideBar/share_common.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_setting">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(245, 247, 250); /* 默认背景颜色 */
    border: none; /* 去掉边框 */
    border-radius: 10px; /* 圆角按钮 */
    padding: 5px;
    color: black; /* 默认字体颜色 */
}

QPushButton:hover {
    background-color: rgb(235, 237, 241); /* 鼠标悬浮时的背景颜色 */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resource/icon/icon.qrc">
           <normaloff>:/SideBar/setting.svg</normaloff>:/SideBar/setting.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/icon/icon.qrc"/>
  <include location="../../resource/icon/icon.qrc"/>
  <include location="../../resource/icon/icon.qrc"/>
  <include location="../../resource/icon/icon.qrc"/>
  <include location="../../resource/icon/icon.qrc"/>
  <include location="../../resource/icon/icon.qrc"/>
  <include location="../../resource/icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
